### PR TITLE
Fix setup documentation

### PR DESCRIPTION
**Pull request by OferLevi85**

### CODE REVIEW

File: docs/local_ui_development.md [hunk 1]
```diff
@@ -55,4 +55,24 @@ npm install nodemon -g
...
 To look at a pre-created "athenz" domain during start up, login with "athenz-admin" as username and password.
 
-Now you can keep the UI instance running in the terminal, make changes to the code using your IDE and UI dev server will reload with your changes.
\ No newline at end of file
+Now you can keep the UI instance running in the terminal, make changes to the code using your IDE and UI dev server will reload with your changes.
+
+##Troubleshooting
+###Clean old artifacts
+It is possible for build to fail if Athenz was previously built. To clean old artifacts:
+1. Run `make make remove-all` from the docker directory
```
Comment (patrasap0908) [inline docs/local_ui_development.md:63]:
```suggestion
1. Run `make remove-all` from the docker directory
```

File: docs/local_ui_development.md [hunk 2]
```diff
@@ -55,4 +55,24 @@ npm install nodemon -g
...
+`security add-trusted-cert -k %s/Library/Keychains/login.keychain ${BASE_DIR}/docker/sample/CAs/athenz_ca.der\n" "$HOME`
+
+If the command didn't work, you may add it using the UI:
+1. Open Keychain Access app
+2. Select File -> Import items...
+3. Select the certificate located at ${BASE_DIR}/docker/sample/CAs/athenz_ca.der
+4. After the certificate is added to the list, click on it
+5. Expand "Trust"
+6. In the field "When using this certificate:" select "Always Trust".
+7. Reload https://localhost
```
Comment (patrasap0908) [inline docs/local_ui_development.md:78]: Maybe also provide a link to https://github.com/AthenZ/athenz/blob/master/docs/setup_ui.md#ui-access if they don't want to add the self-signed cert to their kaychain?

File: docs/local_ui_development.md (hunk 1 above)
Comment (OferLevi85) [reply docs/local_ui_development.md]: Fixed

File: docs/local_ui_development.md (hunk 2 above)
Comment (OferLevi85) [reply docs/local_ui_development.md]: Fixed

**Review by patrasap0908 — APPROVED** [review]

**Comment by abvaidya** [comment]
I think for clean up we can just users to run stop-local-athenz.sh

That also prints an optional make target to delete local images.

**Review by abvaidya — APPROVED** [review]

**Pull request merged**